### ISSUE TITLE
Fix Teamcity inspections issues

### ISSUE THREAD

**Issue by romani** [checkstyle/checkstyle#7309]
from TC - https://teamcity.jetbrains.com/viewLog.html?buildId=2662273&tab=Inspection&buildTypeId=Checkstyle_IdeaInspectionsMaster

> This server has been updated to 2019.2 RC version on 28th of November

>IntelliJ Inspections and Duplicates Engine: Bundled Intellij Idea 2019.3

Violations:
Code style issues
- Constant on the wrong side of comparison (Errors) (2247)

General
-  Duplicated code fragment (30)
-  Redundant suppression (Errors) (8)

JUnit
-  Misordered 'assertEquals()' arguments (Errors) (1)

Verbose or redundant code constructs
-  Redundant Collection operation (1)
-  Redundant type cast (Errors) (7)
-  Unnecessarily escaped character (1)

**Comment by rnveach** [comment]
This is new TC violations that I have seen on my local.

> Constant on the wrong side of comparison (Errors) (2247)

Configuration just needs to be updated to specify what the right side is. TC defaults to constant being on left.

**Comment by pbludov** [comment]
This [violation](https://teamcity.jetbrains.com/viewLog.html?buildId=2662273&tab=Inspection&buildTypeId=Checkstyle_IdeaInspectionsMaster#) is false-positive:
```
        catch (IllegalArgumentException expected) {
            assertEquals("Invalid exception message",
                    "given id " + nextAfterMaxId, expected.getMessage());
        }
```
But the name of the exception variable is a bit confusing.

**Comment by pbludov** [comment]
Java inspection `Duplicates` seems to be renamed to `DuplicatedCode`
inspection `ConstantOnRHSOfComparison` to `ConstantOnWrongSideOfComparison`

**Comment by pbludov** [comment]
Removing cast from `IndentationCheckTest#testArgumentOrderOfErrorMessages` won't work:
```
IndentationCheckTest.java (4)
1981: testArgumentOrderOfErrorMessages() Confusing argument arguments, unclear if a varargs or non-varargs call is desired 
1982: testArgumentOrderOfErrorMessages() Confusing argument arguments, unclear if a varargs or non-varargs call is desired 
1983: testArgumentOrderOfErrorMessages() Confusing argument arguments, unclear if a varargs or non-varargs call is desired 
1984: testArgumentOrderOfErrorMessages() Confusing argument arguments, unclear if a varargs or non-varargs call is desired 
```
https://teamcity.jetbrains.com/viewLog.html?buildId=2662512&tab=Inspection&buildTypeId=Checkstyle_IdeaInspectionsPullRequest

**Comment by romani** [comment]
fix is merged
